index.d.ts

```diff
@@ -1561,6 +1713,9 @@ declare namespace Eris {
     addRelationship(userID: string, block?: boolean): Promise<void>;
     addSelfPremiumSubscription(token: string, plan: string): Promise<void>;
     banGuildMember(guildID: string, userID: string, deleteMessageDays?: number, reason?: string): Promise<void>;
+    bulkEditCommandPermissions(guildID: string, permissions: ApplicationCommandPermissions[]): Promise<GuildApplicationCommandPermissions[]>;
```

Comment:
```suggestion
    bulkEditCommandPermissions(guildID: string, permissions: { id: string; permissions: ApplicationCommandPermissions }[]): Promise<GuildApplicationCommandPermissions[]>;
```
`permissions` type is wrong
Not sure if this should be separated into another type; I think it's short enough though.